File: vivid/src/main/java/com/icetea09/vivid/model/ImageFactory.java
package com.icetea09.vivid.model;

import com.icetea09.vivid.helper.ImagePickerUtils;

import java.util.ArrayList;
import java.util.List;

public class ImageFactory {
    
    public static List<Image> singleListFromPath(String path) {
        List<Image> images = new ArrayList<>();
        images.add(new Image(0, ImagePickerUtils.getNameFromFilePath(path), path, true));
        return images;
    }
}

File: library/src/main/java/com/icetea09/vivid/model/ImageFactory.kt
package com.icetea09.vivid.model

import com.icetea09.vivid.helper.ImagePickerUtils

import java.util.ArrayList

object ImageFactory {

    fun singleListFromPath(path: String): List<Image> {
        val images = ArrayList<Image>()
        images.add(Image(0, ImagePickerUtils.getNameFromFilePath(path), path, true))
        return images
    }
}
